ui/widget.js

```diff
@@ -514,7 +514,7 @@ $.Widget.prototype = {
 				// - disabled class as method for disabling individual parts
 				if ( !suppressDisabledCheck &&
 						( instance.options.disabled === true ||
-							$( this ).hasClass( "ui-state-disabled" ) ) ) {
+						$( this ).hasClass( "ui-state-disabled" ) ) ) {
```

Comment:
Esformatter tends to leave line breaks as-is. I can always update settings to enforce them one way or the other. I'll going with merging this PR for now, without further changes.